### PR TITLE
Added new documentation for external driver

### PR DESCRIPTION
**Pull request by ebarriosjr**
Added documentation for pot driver. Pot is a framework that handles FreeBSD jails. Pot documentation can be found under: https://github.com/pizzamig/pot

### CODE REVIEW

File: website/source/docs/drivers/external/pot-task-driver.html.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+* Installing [pot][pot-github-repo] and following the install [guide][pot-install-guide].
+* Webserver from where to serve the images. (simple file server)
+* Following lines need to be included in your rc.conf
+
+```
+nomad_user="root"
+nomad_env="PATH=/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/sbin:/bin"
+```
+
+[pot-task-driver]: https://github.com/trivago/pot-task-driver
```
Comment (tgross) [inline website/source/docs/drivers/external/pot-task-driver.html.md]: It looks like this was moved and should be https://github.com/trivago/nomad-pot-driver

File: website/source/docs/drivers/external/pot-task-driver.html.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+layout: "docs"
+page_title: "Drivers: pot-task-driver"
+sidebar_current: "docs-drivers-community-pot-task-driver"
+description: |-
+  The Pot task driver is used to run pot (https://github.com/pizzamig/pot) containers using FreeBSD jails.
+---
+
+# Pot task Driver
+
+Name: `pot-task-driver`
```
Comment (tgross) [inline website/source/docs/drivers/external/pot-task-driver.html.md]: The name we use here, as the documentation page title, and table of contents, should match the name that the driver registers (`pot`, in this case). I think it's only `jail-task-driver` that's the oddball here using "task-driver" in the name.

File: website/source/docs/drivers/external/pot-task-driver.html.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+The pot task driver supports the following parameters: 
+
+* `image` - The url for the http registry from where to get the image.
+
+* `pot` - Name of the image in the registry.
+
+* `tag` - Version of the image.
+ 
+* `commad` - Command that is going to be executed once the jail is started.
```
Comment (tgross) [inline website/source/docs/drivers/external/pot-task-driver.html.md]:
```suggestion
* `command` - Command that is going to be executed once the jail is started.
```

File: website/source/docs/drivers/external/pot-task-driver.html.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  **"host"** pot bounds the jail directly to a host port.
+
+* `port_map` - (Optional) Sets the port on which the application is listening inside of the jail. If not set, the application will inherit the port configuration from the image.
+
+* `copy` - (Optional) Copies a file from the host machine to the pot jail in the given directory.
+
+* `mount` - (Optional) Mounts a read/write folder from the host machine to the pot jail.
+
+* `mount_read_only` - (Optional) Mounts a read only directory inside the pot jail.
+
```
Comment (tgross) [inline website/source/docs/drivers/external/pot-task-driver.html.md]: I checked the task config in the source at [driver.go#L51-L66](https://github.com/trivago/nomad-pot-driver/blob/5b02965f3cfc06a28d631043efc4a76987c57ef9/driver/driver.go#L51-L66) and it looks like the `args`, `debug`, and `verbose` flags have been left off.

**Comment by tgross** [comment]
Those changes looks great @ebarriosjr!

One last thing is that the commit 15417d8 was made with a different user than the other commits, so the CLA bot thinks there's a second contributor. If you squash these down then the CLA bot should be happy and I can merge this.

**Comment by ebarriosjr** [comment]
Created new PR with sqashed history.
https://github.com/hashicorp/nomad/pull/6369

**Pull request closed without merging**